sympy/functions/elementary/complexes.py

```diff
@@ -597,17 +605,31 @@ def _eval_rewrite_as_atan2(self, arg):
         x, y = re(self.args[0]), im(self.args[0])
         return atan2(y, x)
 
+
 class conjugate(Function):
     """
-    Changes the sign of the imaginary part of a complex number.
+    Returns the `complex conjugate of an argument. In mathematics, the complex
+    conjugate of a complex number is given by changing the sign of the
+    imaginary part. Thus, the conjugate of the complex number
+    :math:`a + ib` (where a and b are real numbers) is :math:`a - ib`.
 
     Examples
     ========
 
-    >>> from sympy import conjugate, I
-
+    >>> from sympy.functions import conjugate
```

Comment:
Leave that as an import from `sympy`.
Examples are supposed to look like user code, and users should not have to care what module each SymPy name originates from.